src/scss/components/_band.scss

```diff
@@ -1,4 +1,5 @@
 .band {
+  overflow: hidden;
```

Comment:
The only issue I'm seeing - which also appears to be present on Support Us - is that on widths larger than 1150px, the bottom of one of the shapes is cut off:

<img width="337" alt="Screen Shot 2020-12-27 at 12 21 40 PM" src="https://user-images.githubusercontent.com/411808/103178219-23c1ae80-483e-11eb-9db4-557a3e8bb411.png">

But at that page width you can already see the sides of those shapes and things are already a bit goofy anyway, so the fact that this looks great in iOS is well worth it. I wouldn't expect most visitors to this site to be on such a large screen, so we can leave this as is.

Nice work! I hope it was an interesting learning experience debugging through the simulator.

Reply:
Absolutely! Definitely seems like a tool that will be useful for me to be familiar with in the future.